src/game/server/gamemodes/instagib/base_fng.cpp

```diff
@@ -335,7 +335,7 @@ inline void CGameControllerBaseFng::UpdateScoresAndDisplayPoints(CPlayer *pKille
 	pKiller->AddScore(PlayerScore - 1);
 	AddTeamscore(pKiller->GetTeam(), TeamScore);
 	if(pKiller->IsPlaying()) // NOLINT(clang-analyzer-unix.Malloc)
-		MakeLaserTextPoints(pKiller->GetCharacter()->GetPos(), PlayerScore, 3); // NOLINT(clang-analyzer-unix.Malloc)
+		MakeLaserTextPoints(pKiller->GetCharacter()->GetPos(), PlayerScore, 3, pKiller->GetCharacter()->TeamMask()); // NOLINT(clang-analyzer-unix.Malloc)
```

Comment:
This looks unsafe. Are you sure `pKiller->GetCharacter()` is never `nullptr`? Cant you trigger a score while being dead or spectator?

Reply:
Ah yes looks safe lol it is already used in the same line lol, never mind xd

Reply:
Oh IsPlaying guards it. I did not know that method.

```C++
bool CPlayer::IsPlaying() const
{
	return m_pCharacter && m_pCharacter->IsAlive();
}
```